pkgs/wezterm/package.nix

```
@@ -0,0 +1,12 @@
+{ buildCatppuccinPort }:
+
+buildCatppuccinPort {
+  pname = "wezterm";
+
+  dontCatppuccinInstall = true;
+
+  postInstall = ''
+    mkdir -p $out
+    mv *.toml $out/
+  '';
```

I think this will be better, and should allow for the `require("${sources.wezterm}/init.lua").apply_to_config(config)` line to work :)
```suggestion
  installTargets = [ "dist" "plugin" ];
```

---

I did that (and updated paths to the `toml` files and `init.lua`) though now wezterm can't find the init.lua though it is accessible at the location (on nix darwin). No idea why.

---

ahh, sorry that would be partly my fault in getting used to the new package system. you will have to specify `dist/path-to.toml` and `plugin/init.lua` instead of what we have currently

---

but it seems you figured that out

---

ahh the user would also have to add `return config`, which i left out incase a user wanted to add more code.